start

```diff
@@ -33,16 +45,30 @@ done
 
 cd "${projectRoot}"
 
-./ffc-sfd-landing-page/scripts/start -d;
-./ffc-sfd-auth/scripts/start -d;
-./ffc-sfd-business/scripts/start -d;
-./ffc-sfd-data/scripts/start -d;
-./ffc-sfd-permissions/scripts/start -d;
-./ffc-sfd-permissions-api/scripts/start -d;
-./ffc-sfd-ahwp-stub/scripts/start -d;
-./ffc-sfd-messages/scripts/start -d;
-./ffc-sfd-messages-processor/scripts/start -d;
-./ffc-sfd-customer-receiver-messages/scripts/start -d;
-./ffc-sfd-customer-receiver-payments/scripts/start -d;
-./ffc-sfd-customer-receiver-preferences/scripts/start -d;
-./ffc-sfd-proxy/scripts/start -d; # Must be after all web applications
+if [ "${sfd}" = "true" ]; then
+  ./ffc-sfd-data/scripts/start -d;
+  echo "Waiting for cosmosDB to start"
```

Comment:
Is it not misleading to have this here as it might not necessarily be the case if cosmos container was already running if it was created by receivers?

Also now means log output from dependency is tightly coupled here.

If we want to explain that cosmos is starting suggest we log that during hte `ffc-sfd-data` startup script itself.

Reply:
Removed it